core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/CloneWorkspaceFileTransferMonitor.scala

```diff
@@ -41,6 +41,9 @@ class CloneWorkspaceFileTransferMonitorActor(val dataSource: SlickDataSource, va
       _ <- pendingTransfers.toList.traverse { pendingTransfer =>
         IO.fromFuture(IO(copyBucketFiles(pendingTransfer))).attempt.map {
           case Left(e) => {
+            // We do not want to throw e here. traverse stops executing as soon as it encounters a Failure, but we
```

Comment:
Added this comment here as well since the reasoning behind not throwing the exception here is identical to WorkspaceBillingAccountMonitor